sw/simulation/agents/particle/particle.cpp

```diff
@@ -22,13 +22,16 @@ std::vector<float> particle::state_update(std::vector<float> state)
   controller->get_velocity_command(ID, v_x, v_y);
   controller->saturate(v_x);
   controller->saturate(v_y);
-  moving = controller->moving;
-  happy = controller->happy;
+  terminalinfo::debug_msg("V_x: "+std::to_string(v_x));
+  terminalinfo::debug_msg("V_y: "+std::to_string(v_y));
```

Comment:
Same as the particle_oriented comments, leave this be and make a new one if you need to (and no need to push it to master unless it's final)